chroma_core/services/rpc.py

```diff
@@ -349,9 +362,19 @@ def _send(self, connection, request):
         """
         log.debug("send %s" % request['request_id'])
         request['response_routing_key'] = self._response_routing_key
+
+        def errback(exc, _):
```

Comment:
Both errback and retry_policy appear to be deplicated. Is it possible to define them once?

Reply:
It's possible, but don't think it buys much